media-gfx/alembic/metadata.xml

```diff
@@ -14,9 +14,25 @@
 	developed by Sony Pictures Imageworks Inc. and Lucasfilm Ltd.
 </longdescription>
 <use>
+	<flag name="arnold" restrict="&gt;=media-gfx/alembic-1.7.9">
+		Add support for arnold renderer (experimental)
+	</flag>
 	<flag name="boost">Use <pkg>dev-libs/boost</pkg> library</flag>
+	<flag name="examples">Install alembic examples</flag>
 	<flag name="hdf5">Add support <pkg>sci-libs/hdf5</pkg></flag>
-	<flag name="pyalembic">Build pyalembic python bindings (needs <pkg>dev-python/pyilmbase</pkg>)</flag>
+	<flag name="maya" restrict="&gt;=media-gfx/alembic-1.7.9">
+		Compile Alembic plugin for Maya (experimental)
+	</flag>
+	<flag name="prman" restrict="&gt;=media-gfx/alembic-1.7.9">
+		Compile Alembic plugin for Renderman (experimental)
+	</flag>
+	<flag name="pyalembic" restrict="&lt;media-gfx/alembic-1.7.9">
```

Comment:
same as above

Reply:
So with this one and the next one (pyalembic and python use flags) hasn't the meaning actually changed by renaming the first into the latter?